src/views/bootablevolumes/list/hooks/useBootableVolumesFilters.ts

```diff
@@ -0,0 +1,45 @@
+import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { OS_NAMES } from '@kubevirt-utils/resources/template';
+import { FilterValue, RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+import { getPreferenceOSType } from '../../utils/utils';
+
+type ItemsToFilterProps = {
+  id: string;
+  title: string;
+};
+
+// return the name or 'Other' if the name not included in the array of available items for filtering
+const getItemNameWithOther = (itemName: string, items: ItemsToFilterProps[]): string => {
+  return !items?.find((s: ItemsToFilterProps) => s.id === itemName) || itemName === 'Other'
+    ? 'Other'
+    : itemName;
+};
+
+const includeFilter = (
+  compareData: FilterValue,
+  items: ItemsToFilterProps[],
+  itemName: string,
+): boolean => {
+  const compareString = getItemNameWithOther(itemName, items);
+
+  return compareData.selected?.length === 0 || compareData.selected?.includes(compareString);
+};
```

Comment:
can be exported to util file?

Reply:
Do you mean util file for _bootablevolumes_ dir, or something more general?